CONTRIBUTING.md

```diff
@@ -17,6 +17,20 @@ When making a feature request, please make it clear what problem you intend to
 solve with the feature, any ideas for how `transit_model` could support solving
 that problem, any possible alternatives, and any disadvantages.
 
+## Pull Request
+
+If you feel to directly submit a PR, please ensure to explain your motivations
+in your PR's description. See also the section below about updating versions of
+crates.
+
+### Update version
+
+The PR owner is the best to understand which evolution and breaking 
```

Comment:
it feels a bit awkward to bump the version if the previous has not been released no?
I’d understand it if this was continuously released but it does not seems the case.
Isn’t it better to do it at release time ?

Reply:
> it feels a bit awkward to bump the version if the previous has not been released no?

Yes, and maybe it's not clear that this is exactly what I'm trying to explain here, but you should not bump the version if it's already alright.

Examples:
1. Version 1.0.0 is already released and `Cargo.toml` contains `1.0.0`
2. Alice makes a bug fix, she bumps the `Cargo.toml` to `1.0.1`
3. Bob makes another bug fix; he sees `1.0.1` is not released yet, so he doesn't need to make a change
4. Carl makes a breaking change, he bumps to `2.0.0`
5. Donald makes a new feature; he sees there is already a major bump, so it does not need to bump the version.

And notice how at each step, we could have published a release right away.

>  Isn’t it better to do it at release time ?

The pain to do that at release time is that the owner of the release doesn't necessarily know the details of each PR (is it a breaking change or not?), and would need to either do a new PR only to bump a version (that's a shame for just a version bump) or push a new commit directly to `master` (not a fan of pushing commits directly to `master` and actually, `master` might just be protected against that).

Reply:
hum isn’t it  difficult for Bob to know that there has not been a release?

Reply:
https://crates.io/crates/transit_model/versions should be enough.

Reply:
IMO, instruct to compare with crates.io is the clearest you can do here, I'd add it to the sentence :)

Reply:
I've introduced it as a link, tell me if that's fine.